internal/pkg/cli/flag.go

```diff
@@ -112,6 +122,16 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 
 	storageFlagDescription        = "Name of the storage resource to create."
 	storageServiceFlagDescription = "Name of the service to associate with storage."
+
+	numFlagDescription              = "Optional. The number of tasks to set up. Default to 1."
+	cpuFlagDescription              = "Optional. The number of cpu-units to reserve. Default to 256."
```

Comment:
Nit: Does CPU units warrant more explanation? I know I still never really get what this means =P